### DISCUSSION TITLE
Rating categories

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/RepoRater#23] [Ideas]
Currently we have a single rating on DX, but we could have other categories also

Ideas...
- how easy to get the project setup
- how straightforward to contribute to the project
- how friendly is the community
- ...

**Comment by CBID2** [comment]
How about these categories @eddiejaoude:
1. Ease of Use
2. How effective the documentation is
3. How prompt code reviews are

**Reply by j-grosse** [reply]
effective communication
